autosetup.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# global variables
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+RESET_PRINT='\033[0;0m'
+
+installed_packages="0"
+required_packages="0"
+venv_name="venv_sapsam"
+kernel_name="sapsam_kernel"
+
+# checks for pip
+required_packages=$((required_packages + 1))
+if pip --version &>/dev/null; then
+    echo -e "\xE2\x9C\x94 pip installed..."
+    installed_packages=$((installed_packages + 1))
+else
+    echo "pip not installed"
+    echo "try installing pip with for example (on macOS): brew install pip"
+    echo "exiting setup script..."
+    exit 1
+fi
+
+# checks for python3
```

Comment:
What if `python3` runs as `python`?

Reply:
Added a statement to check for the python alias for python3. Although my worry is, what happens if the user doesn't have python3 installed? The minimum version required for the project is 3.8.13, according to environment-lock.yml